src/support.njk

```diff
@@ -1,27 +1,114 @@
 ---
-layout: layouts/nohero.njk
-title: Request Support
+align: center
+layout: layouts/page.njk
+title: We’re here to help!
```

Comment:
Is the title element in the HEAD of the HTML the same? In that case I'd rather have Support

Reply:
Yeah, it's the same... Will change it back to Support

Reply:
Help center is fine with me